flytekit/image_spec/image_spec.py

```diff
@@ -127,6 +131,17 @@ def __post_init__(self):
                 error_msg = f"{parameter} must be a list of strings or None"
                 raise ValueError(error_msg)
 
+        parameters_two_str_tuple_list = ["pip_secret_mounts"]
+        for parameter in parameters_two_str_tuple_list:
+            attr = getattr(self, parameter)
```

Comment:
Given that this is the only parameter that accepts a list of tuples of strings, can we simplify this validation:

```python
        if self.pip_secret_mounts is not None:
            ...
```